### ISSUE TITLE
is there any chance to write captions on multiple lines if they are too long, using CSS or JS?

### ISSUE THREAD

**Issue by gzoidze** [Spyna/react-svg-radar-chart#20]

**Comment by gzoidze** [comment]
@Spyna

**Comment by Spyna** [comment]
Hi, 
the caption are rendered in a `<text>` SVG component: https://developer.mozilla.org/en-US/docs/Web/SVG/Element/text. 
You can use the `captionProps` to customize it. 

A solution could be something like this: 

```
 options: {
   //  ...other options
    captionProps : (column) => ({
        className: 'custom-caption', //a CSS class to customize the text element
        onMouseOver : this.handleToolTip, // a function that shows a tooltip on mouse over
      })
  },
```

**Comment by gzoidze** [comment]
I still can't make it work. if I have about 10 captions names are overlapped. lots of CSS properties do not work to write names on multiple lines.

**Comment by gabrilator** [comment]
Any news regarding this? 
Thanks so much :)

**Comment by Spyna** [comment]
> f CSS pr

in this case you can add some margin using: 

```
 options : {
      captionMargin: 30,
 }
```
the default value for the margin is 10, so

**Comment by Spyna** [comment]
I've added these new options: 
```
wrapCaptionAt: 15
captionLineHeight: 20
```
The first is the max length of the caption before it gets wrapped in a new line, and the second is the line-height of the caption when it's wrapped. 

Playing with these two options should fix this issue. 

The options are available form version 1.4.0
